modules/scrabble.js

```diff
@@ -112,10 +112,11 @@ module.exports = {
       let words = []
       phrase.forEach((word) => {
         word = word.replace(/[^A-Z]+$/, '')
-        if (wordHistory.includes(word) || word === '') return
-        words.push(computeWord(word))
+        let result = computeWord(word)
+        if (wordHistory.includes(word) || !result.isPossible) return
```

Comment:
this should probably be `result.word` because line 121 is
(otherwise the various transforms could be inconsistent)

thinking about it, you don't need the `.toUpperCase()` on line 111 anymore?